flask-backend/migrations/versions/989dbc01a9b0_.py

```diff
@@ -0,0 +1,87 @@
+"""empty message
```

Comment:
Do we need to push this?

Reply:
Yes, it's a part of migrations file, that contains versions of database migrations.